lib/orchestrator/process_background_queue.rb

```diff
@@ -0,0 +1,47 @@
+module Orchestrator
+  class ProcessBackgroundQueue
+    include Mandate
+
+    MAX_QUEUE_LENGTH = 10
+
+    def call
+      $stdout.sync = true
+      $stderr.sync = true
+
+      should_exist = false
+      %w[INT TERM].each do |sig|
+        trap sig do
+          puts "Exit signal recieved"
+          should_exist = true
+        end
+      end
+
+      loop do
+        break if should_exist
+
+        sleep(1)
+        execute
+      rescue StandardError => e
+        # If stuff goes wrong should we abort the machine
+        # or should we just give it a shot. Maybe have a counter?
+        puts e
```

Comment:
Will we somehow be notified if something goes wrong?

Reply:
No :(

I'll add an issue for adding bugsnag to this.